evalml/automl/data_splitters/training_validation_split.py

```diff
@@ -5,7 +5,7 @@
 class TrainingValidationSplit(BaseCrossValidator):
     """Split the training data into training and validation sets"""
 
-    def __init__(self, test_size=None, train_size=None, shuffle=True, stratify=None, random_state=0):
+    def __init__(self, test_size=None, train_size=None, shuffle=False, stratify=None, random_state=0):
```

Comment:
Should this be `shuffle=True`?

Reply:
@gsheni I updated this to be the same as the other sklearn-defined CV methods, which have `shuffle=False` by default. Then, in the automl code which calls this, I set `shuffle=True`. I just did it to be consistent with sklearn.